### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by MeuhV1**
If year is after company fall date and that company is removed from all cities, it skips the cityValue calculations and the spreading/disappearing of companies
Improved canRespawn() function in RFCUtils by adding more return statements
Added pedia jumps to routes page in Improvements and Resources pedia
Removed sensibility check in WB DoC maps editor (was not used anyway)
Some other small fixes/updates

### CODE REVIEW

File: Assets/Python/Pedia/CvPediaImprovement.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def placeDetails(self):
 				if iYieldChange > 0:
 					szSign = u"+"
 				if iYieldChange != 0:
-					szText += u"%c%s%d%c with <link=CONCEPT_MOVEMENT>%s</link>\n" % (szBullet, szSign, iYieldChange, gc.getYieldInfo(iYieldType).getChar(), gc.getRouteInfo(iRoute).getDescription())
```
Comment (dguenms) [inline Assets/Python/Pedia/CvPediaImprovement.py:196]: 👍

File: Assets/Python/RFCUtils.py [hunk 2]
```diff
@@ -1573,25 +1572,26 @@ def canRespawn(self, iPlayer):
 			for tInterval in tResurrectionIntervals[iPlayer]:
 				iStart, iEnd = tInterval
 				if getTurnForYear(iStart) <= iGameTurn <= getTurnForYear(iEnd):
-					bPossible = True
 					break
+			else:
```
Comment (dguenms) [inline Assets/Python/RFCUtils.py:1576]: ... is that an else block after a for loop?

Comment (MeuhV1) [reply Assets/Python/RFCUtils.py:1576]: Yes. Python has a else clause for for loops.
http://book.pythontips.com/en/latest/for_-_else.html

The else activates when the for loops completes normally. If the loop is terminated with a break statement, the else clause does not activate.

Comment (dguenms) [reply Assets/Python/RFCUtils.py:1576]: That's really cool, I learned something today. And it's compatible with Python 2.4.

**Pull request merged**